cmd/minikube/cmd/start.go

```diff
@@ -486,6 +497,16 @@ func generateConfig(cmd *cobra.Command, k8sVersion string) (cfg.Config, error) {
 	return cfg, nil
 }
 
+// noneNeedsResolveConf checks extra config option for resolv.conf is needed
+// for none driver. more info https://github.com/kubernetes/minikube/issues/3511
+func noneNeedsResolveConf() bool {
+	filename := "/run/systemd/resolve/resolv.conf"
```

Comment:
Could this be refactored in a way that doesn't require this filename to be duplicated by the caller?

Perhaps this function could return the extra flag required (kubelet.resolv-conf=xxx), or the value could be shared as a const?